pkgs/tools/text/ispell/default.nix

```diff
@@ -6,25 +6,11 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "http://fmg-www.cs.ucla.edu/geoff/tars/${pname}-${version}.tar.gz";
-    sha256 = "1d7c2fqrdjckp91ajpkn5nnmpci2qrxqn8b6cyl0zn1afb9amxbz";
+    sha256 = "1hmfnz55qzfpz7lz0r3m4kkv31smir92ks9s5l1iiwimhr2jxi2x";
   };
 
   buildInputs = [ bison ncurses ];
 
-  patches = [
-    ./patches/0007-Use-termios.patch
-    ./patches/0008-Tex-backslash.patch
-    ./patches/0009-Fix-FTBFS-on-glibc.patch
-    ./patches/0011-Missing-prototypes.patch
-    ./patches/0012-Fix-getline.patch
-    ./patches/0013-Fix-man-pages.patch
-    ./patches/0021-Fix-gcc-warnings.patch
-    ./patches/0023-Exclusive-options.patch
-    ./patches/0024-Check-tempdir-creation.patch
-    ./patches/0025-Languages.patch
-    ./patches/0030-Display-whole-multibyte-character.patch
```

Comment:
Some of these still might make sense when updated from https://salsa.debian.org/debian/ispell/tree/debian/3.4.00-8/debian/patches. Why they did not upstream them even after all these years is beyond me.

Or if this is intended only for the German dictionary we might make it local as [originally suggested](https://github.com/NixOS/nixpkgs/pull/36155#issuecomment-369531974).

Or maybe we are fine with vanilla ispell, even though some of those patches look like they should definitely be upstreamed.